Modules/Devices/src/Devices.Infrastructure/Persistence/IServiceCollectionExtensions.cs

```diff
@@ -55,6 +55,7 @@ public static void AddDatabase(this IServiceCollection services, Action<DbOption
     public static void AddRepositories(this IServiceCollection services)
     {
         services.AddTransient<IIdentitiesRepository, IdentitiesRepository>();
+        services.AddTransient<ITierRepository, TierRepository>();
```

Comment:
The correct name should be `ITiersRepository`/`TiersRepository`. You can fix that in the next PR.